### PR TITLE
update stateful function docs

### PR DESCRIPTION
**Pull request by tpiperatgod**
Update configuration instructions for stateful functions, service validation, etc.

Signed-off-by: laminar <tpiperatgod@gmail.com>

### CODE REVIEW

**Review by freeznet — APPROVED** [review]

File: docs/functions/run-function/run-stateful-function.md [hunk 1]
```diff
@@ -31,55 +31,124 @@ After packaging a stateful function, you can submit it to a Pulsar cluster. This
...
-    ### gRPCServer ###
+    Under the namespace (`${PULSAR_NAMESPACE}`) where the pulsar cluster is installed, find the ConfigMap resource named `${PULSAR_RELEASE_NAME}-pulsar-bookie` ( in which `${PULSAR_RELEASE_NAME}` is the release name when the pulsar cluster is installed using helm chart) and edit it.
 
-    # The gRPC server port to listen on. The default is 4181.
-    storageserver.grpc.port=4181
+    ```shell
+    kubectl edit -n ${PULSAR_NAMESPACE} ${PULSAR_RELEASE_NAME}-pulsar-bookie
+    ```
+
+    Then insert the configurations in the following location.
```
Comment (Huanli-Meng) [inline docs/functions/run-function/run-stateful-function.md:77]:
```suggestion
    Insert the configurations in the following location.
```

**Review by Huanli-Meng — COMMENTED** [review]
Leave my comments. BTW, do we need to update docs for v0.3.0?

**Comment by tpiperatgod** [comment]
> Leave my comments. BTW, do we need to update docs for v0.3.0?

done.

yes, because the stateful function has been supported since v0.3.0, we need to update the docs for v0.3.0.

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Pull request merged**